calico/getting-started/kubernetes/self-managed-onprem/onpremises.md

```diff
@@ -11,18 +11,16 @@ Install {{site.prodname}} to provide both networking and network policy for self
 
 **{{site.prodname}} networking** and **network policy** are a powerful choice for a CaaS implementation. If you have the networking infrastructure and resources to manage Kubernetes on-premises, installing the full {{site.prodname}} product provides the most customization and control.
 
-### Features
-
-This how-to guide uses the following {{site.prodname}} features:
+### Concepts
 
-- **calico/node**
-- **Typha**
+#### {{site.prodname}} operator
 
-### Concepts
+{{site.prodname}} is installed by an operator which manages the installation, upgrade, and general lifecycle of a {{site.prodname}} cluster. The operator is
+installed directly on the cluster as a Deployment, and is configured through one or more custom Kubernetes API resources.
 
 #### {{site.prodname}} manifests
 
-{{site.prodname}} provides manifests for easy customization. Each manifest contains the necessary resources for installing {{site.prodname}} on each node in your Kubernetes cluster. You may want to [customize the {{site.prodname}} manifests]({{site.baseurl}}/getting-started/kubernetes/installation/config-options) before installing {{site.prodname}} on nodes.
+{{site.prodname}} can also be installed using raw manifests as an alternative to the operator. The manifests contain the necessary resources for installing {{site.prodname}} on each node in your Kubernetes cluster. Using manifests is not recommended as they cannot automatically manage the lifecycle of the {{site.prodname}} as the operator does. However, manifests may be useful for clusters that require highly specific modifications to the underlying Kubernetes resources.
```

Comment:
Maybe also mention that at upgrade time, someone will need to repeat the customisation process on the uplevel manifest, whereas with Operator that's all taken care of.